### PR TITLE
Change the script in the tutorial

### LINKED ISSUE #950: Edit in the User Guide Chap 1-7
--------
Issue Description
--------
User guide reference scripts in all chapters generate figures with the wrong colors(All figures).

Reference subchapters cannot be linked in a different chapter. (P.54 p3 ...) 

The batch size number in the text is different from the number in the script(P.55 p2 ...).

Some description is confusing (P55 " run **simultaneously.** Every time a simulation ends, a new one is launched.") and redundant (P25 line 1 and line 2)
 
Some reference scripts have the unused subclass (P52 <Files> ...output_1... output2...</Files>)

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or task?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by JiaZhou-PU**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Closes #950 

##### What are the significant changes in functionality due to this change request?
From Chapter 1-7

Fix color command in the user guide reference scripts and figures in gold Dir.

Fix cross reference subchapters.

Fix batch size number in the text. Fix incorrect equations and format.

Delete confusing description, and redundant ones as well.  Delete unused reference script's objects.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

**Comment by PaulTalbot-INL** [comment]
FYI, I think the conflicts are coming from #941, which was merged after you started working on the user guide.

**Comment by JiaZhou-PU** [comment]
@PaulTalbot-INL ready for review.

File: doc/user_guide/forwardSamplingExample.tex [hunk 1]
```diff
@@ -51,10 +51,10 @@ \subsection{Monte-Carlo sampling through RAVEN}
 \begin{enumerate}
    \item \textbf{\textit{RunInfo}}:
      \xmlExample{framework/user_guide/ForwardSamplingStrategies/forwardSamplingMontecarlo.xml}{RunInfo}
-     As discussed in Section~\ref{sub:EntitiesAndFlow}, the \textit{RunInfo} \textbf{Entity} sets up the analysis
+     As discussed in Section~\ref{sub:singleRun}, the \textit{RunInfo} \textbf{Entity} sets up the analysis
      that the user wants to perform. In this specific case, two steps (\xmlNode{Sequence}) are sequentially run
-     using 12 processors (\xmlNode{batchSize}). This means that
-     12 instances of the driven code are run simultaneously. Every time a simulation ends, a new one is launched.
+     using 1 processor (\xmlNode{batchSize}). If this number change into 12, this means that
```
Comment (PaulTalbot-INL) [inline doc/user_guide/forwardSamplingExample.tex]: maybe "If this number were changed to 12"?

File: doc/user_guide/forwardSamplingExample.tex [hunk 2]
```diff
@@ -314,11 +313,10 @@ \subsection{Stratified sampling through RAVEN}
...
    \item \textbf{\textit{RunInfo}}:
      \xmlExample{framework/user_guide/ForwardSamplingStrategies/forwardSamplingStratified.xml}{RunInfo}
-   As reported in Section~\ref{sub:EntitiesAndFlow}, the \textit{RunInfo} \textbf{Entity} is intended to set up the analysis
+   As reported in Section~\ref{sub:singleRun}, the \textit{RunInfo} \textbf{Entity} is intended to set up the analysis
    that the user wants to perform. In this specific case, two steps (\xmlNode{Sequence}) are  run
-   using 12 processors (\xmlNode{batchSize}). This means that
-   12 instances of the driven code are run simultaneously.
-   Every time a simulation ends, a new one is launched.
+   using 1 processor (\xmlNode{batchSize}). This means that
+   1 instance of the driven code are run simultaneously.
```
Comment (PaulTalbot-INL) [inline doc/user_guide/forwardSamplingExample.tex]: are -> is, but the sentence itself doesn't make sense with a single processor. Maybe "This means that a single instance of the driven code is run at a time."

File: doc/user_guide/ravenPostProcess.tex [hunk 3]
```diff
@@ -65,7 +65,7 @@ \subsection{Build RAVEN Input: \xmlNode{PostProcess}}
      collect the outputs in the two \textit{DataObjects}. As it can be seen, the \xmlNode{Sampler} is inputted
      to communicate to the \textit{Step} that the driven code needs to be perturbed through the Monte Carlo sampling
      strategy.
-  \item \xmlNode{PostProcess} named ``statAnalysis\_1'', is used to compute all the statistical moments and FOMs
+  \item \xmlNode{PostProcess} named ``statAnalysis\_1'', is used to compute all the statistical moments and FOMs
```
Comment (PaulTalbot-INL) [inline doc/user_guide/ravenPostProcess.tex]: trailing whitespace; you may be able to set your editor up to automatically remove this.

File: doc/user_guide/singleRunAndPlot.tex [hunk 4]
```diff
@@ -173,7 +173,6 @@ \subsection{Build RAVEN Input: \xmlNode{IOStep}}
 The \textbf{IOStep} can be used to:
 \begin{itemize}
   \item construct/update a \textit{Database} from a \textit{DataObjects} object, and vice versa;
-  \item construct/update a \textit{DataObject} from a \textit{CSV} file contained in a directory;
```
Comment (PaulTalbot-INL) [inline doc/user_guide/singleRunAndPlot.tex]: Why was this line removed?

File: scripts/establish_conda_env.sh [hunk 5]
```diff
@@ -80,7 +80,7 @@ function find_conda_defs ()
       then
         CONDA_DEFS="/c/ProgramData/Miniconda2/etc/profile.d/conda.sh";
       else
-        CONDA_DEFS="$HOME/miniconda2/etc/profile.d/conda.sh";
+        CONDA_DEFS="$HOME/miniconda3/etc/profile.d/conda.sh";
```
Comment (PaulTalbot-INL) [inline scripts/establish_conda_env.sh]: this shouldn't be changed globally; this is particular to a build. I would recommend reverting this for now; it should be taken care of in a separate merge request when we make python3 the default.

File: tests/framework/user_guide/ForwardSamplingStrategies/forwardSamplingSparseGrid.xml [hunk 6]
```diff
@@ -181,17 +181,21 @@
       <Sampler class="Samplers" type="SparseGridCollocation">SG</Sampler>
       <Output class="DataObjects" type="HistorySet">histories</Output>
     </MultiRun>
+    <RomTrainer name="train">
+      <Input class="DataObjects" type="HistorySet">histories</Input>
+      <Output class="Models" type="ROM">rom</Output>
+    </RomTrainer>
```
Comment (PaulTalbot-INL) [inline tests/framework/user_guide/ForwardSamplingStrategies/forwardSamplingSparseGrid.xml]: Why this change? It's not a problem, just curious.

File: tests/framework/user_guide/ravenTutorial/PostProcess.xml [hunk 7]
```diff
@@ -33,8 +33,8 @@
     <Input name="referenceInput.xml" type="input">
         ../commonFiles/referenceInput_generic_CI.xml
     </Input>
-    <Input name="output_1" type="">static.xml</Input>
-    <Input name="output_2" type="">time.xml</Input>
+    <!--Input name="output_1" type="">static.xml</Input>
+    <Input name="output_2" type="">time.xml</Input-->
```
Comment (PaulTalbot-INL) [inline tests/framework/user_guide/ravenTutorial/PostProcess.xml]: we can remove these commented lines

File: doc/user_guide/singleRunAndPlot.tex (hunk 4 above)
Comment (JiaZhou-PU) [reply doc/user_guide/singleRunAndPlot.tex]: Duplicated from the last bullet

File: scripts/establish_conda_env.sh (hunk 5 above)
Comment (JiaZhou-PU) [reply scripts/establish_conda_env.sh]: Fixed, change back into the original one

File: tests/framework/user_guide/ForwardSamplingStrategies/forwardSamplingSparseGrid.xml (hunk 6 above)
Comment (JiaZhou-PU) [reply tests/framework/user_guide/ForwardSamplingStrategies/forwardSamplingSparseGrid.xml]: Just want to get a better understanding of the steps, I put this in sequence order, even though it does not to.

File: tests/framework/user_guide/ravenTutorial/PostProcess.xml (hunk 7 above)
Comment (JiaZhou-PU) [reply tests/framework/user_guide/ravenTutorial/PostProcess.xml]: I will delete this.

File: doc/user_guide/forwardSamplingExample.tex (hunk 1 above)
Comment (JiaZhou-PU) [reply doc/user_guide/forwardSamplingExample.tex]: Changed

File: doc/user_guide/forwardSamplingExample.tex (hunk 2 above)
Comment (JiaZhou-PU) [reply doc/user_guide/forwardSamplingExample.tex]: Changed

File: doc/user_guide/ravenPostProcess.tex (hunk 3 above)
Comment (JiaZhou-PU) [reply doc/user_guide/ravenPostProcess.tex]: Deleted

File: doc/user_guide/singleRunAndPlot.tex (hunk 4 above)
Comment (JiaZhou-PU) [reply doc/user_guide/singleRunAndPlot.tex]: Keep this one

File: tests/framework/user_guide/ravenTutorial/PostProcess.xml (hunk 7 above)
Comment (JiaZhou-PU) [reply tests/framework/user_guide/ravenTutorial/PostProcess.xml]: Deleted

**Comment by JiaZhou-PU** [comment]
checkout the devel scripts/establish_conda_env.sh
Ready for review again @PaulTalbot-INL
